sync/storage/batch/batch_test.go

```diff
@@ -140,19 +140,19 @@ func TestSync(t *testing.T) {
 						Return([]*models.BucketDescriptor{bucket1, bucket2}, nil).
 						Times(1),
 					c.EXPECT().
-						ListSourceFilesAsc(gomock.Any(), bucket1.Name).
+						ListSourceFilesAsc(gomock.Any(), bucket1.Name, time3).
 						Return([]*models.FileDescriptor{file1V3, file2V2}, nil).
 						Times(1),
 					c.EXPECT().
-						ListSourceFilesAsc(gomock.Any(), bucket2.Name).
+						ListSourceFilesAsc(gomock.Any(), bucket2.Name, time3).
```

Comment:
invalid operation: bucket2 (variable of type *invalid type) has no field or method Name